packages/eas-cli/src/utils/profiles.ts

```diff
@@ -47,6 +57,45 @@ export async function getProfilesAsync<T extends ProfileType>({
   return await Promise.all(results);
 }
 
+function isBuildProfile(profile: any, type: ProfileType): profile is BuildProfile<Platform> {
+  return !!profile && type === 'build';
+}
+
+async function setNodeVersionFromFileAsync(
+  projectDir: string,
+  profile: BuildProfile<Platform>
+): Promise<void> {
+  const nodeVersion = await getNodeVersionFromFileAsync(projectDir);
+  if (nodeVersion) {
+    if (profile.node && profile.node !== nodeVersion) {
+      throw new Error(
+        `Node version defined in eas.json (${profile.node}) does not match the version defined in .nvmrc (${nodeVersion})`
+      );
+    }
```

Comment:
I don't think that we want to throw errors here.

I believe that the logic here should be as follows:
1. If the `node` version is specified in `eas.json` just use this value (even though somebody has a `.nvmrc` file)
2. If the `node` version is not specified in `eas.json` and somebody has a `.nvmrc` file use version from the `.nvmrc` file as an override (be vocal about it)